en/02_Developer_Guides/18_Cookies_And_Sessions/02_Sessions.md

```diff
@@ -100,9 +100,52 @@ including form and page comment information. None of this is vital but `clear_al
 $session->clearAll();
 ```
 
-## Cookies
+## Configuration
 
-### Samesite attribute
+### Session lifetime
+
+By default, the session cookie will only last until the user closes their browser, and the session on the server-side will live for the number of seconds set in the [`session.gc_maxlifetime`](https://www.php.net/manual/en/session.configuration.php#ini.session.gc-maxlifetime) ini configuration since it was last touched.
+
+You can configure the session lifetime by setting the [`Session.timeout`](SilverStripe\Control\Session->timeout) configuration property to the number of seconds the session should be valid for since the last modified time.
+
+```yml
+SilverStripe\Control\Session:
+  timeout: 86400
+```
+
+> [!WARNING]
+> If you use a save handler which isn't included in `silverstripe/framework`, it may treat the session lifetime differently. You should check its docuemntation or implementation to understand how it works.
+
+### Save handler
+
+You can choose how session are handled by setting the [`Session.save_handler`](api:SilverStripe\Control\Session->save_handler) configuration property. By default this is set to [`FileSessionHandler`](api:SilverStripe\Control\SessionHandler\FileSessionHandler).
+
+If you want to use the default PHP file session handler instead, you can set the `Session.save_handler` configuration to `null`.
+
+```yml
+SilverStripe\Control\Session:
+  save_handler: null
+```
+
+You can also set it to the FQCN or injector service name of any session handler that implements [`SessionHandlerInterface`](https://www.php.net/manual/en/class.sessionhandlerinterface.php).
+
+#### `FileSessionHandler`
+
+The default file-based session handler for PHP holds a lock on the session file while the session is open. In practice this means that multiple concurrent requests from the same user have to wait for one another to finish processing after a session has been started. This includes AJAX requests.
```

Comment:
```suggestion
The default file-based session handler for PHP holds a lock on the session file while the session is open. In practice this means that multiple concurrent requests from the same user have to wait for one another to finish processing after a session has been started. This includes XHR requests.
```

Reply:
AJAX technically only refers to XHR requests that return HTML, wheras we also care about requests that return JSON

Reply:
> AJAX technically only refers to XHR requests that return HTML

I don't think that's true? Do you have some documentation for that? I know the term does stand for "Asynchronous JavaScript And XML", but its usage everywhere I can find has it just meaning essentially "an HTTP request fired off from client-side JavaScript".

Reply:
Opps, apologies got ajax confused with pjax.  Yes ajax is the superset of xhr so is correct.